workers/tests/integrationtests/indicators/test_building_completeness.py

```diff
@@ -105,7 +105,7 @@ def test_get_hex_cells(self, feature):
         assert isinstance(result, FeatureCollection)
         assert result.features is not None
 
-    def test_get_hex_cells_not_found(self, feature):
+    def test_get_hex_cells_not_found(self):
         feature = get_geojson_fixture("heidelberg-altstadt-feature.geojson")

```

Comment:
If a fitting fixtures is already defined in `conftest.py`, it is better to use it. In this case, any geojson outside Africa should work

```suggestion
    def test_get_hex_cells_not_found(self, feature_germany_heidelberg):
```